### DISCUSSION TITLE
Keycloak identity provider(OIDC)

### DISCUSSION THREAD

**Discussion by thesardorbroo** [keycloak/keycloak#20565] [Q&A]
Hi, I have 2 questions. 

1. I'm using **Keycloak** identity provider function and I've integrated Keycloak to **UAE Pass** app, It's working). Keycloak requests to userinfo URL(sets UAE Pass token to header) and Keycloak has caught user's info and saves the info and generates Keycloak token. I should catch the UAE Pass token and set it to Keycloak token's claim. But I can't(. What should I do?

2. There are username field(input), password field(input) and UAE Pass button(identity provider) in my keycloak login page. I should authenticating with UAE Pass with Postman(Rest API). I can get token with username and password but I can't send request to authenticating with UAE Pass. What should I do?

Thanks for attention)

**Comment by donarmando89** [comment]
@thesardorbroo  In order to get valid suggestions on how to resolve the issue, it would be better to provide step-by-step configuration with screenshots on how to configure UAE Pass as an IDP on Keycloak and also share details of the execution steps to reproduce the issue you are encountering. 
Thanks.
